src/components/Match/matchColumns.jsx

```diff
@@ -541,11 +528,7 @@ export const inflictorsColumns = [{
     field: 'damage_inflictor',
     displayFn: (row, col, field) => (field ? Object.keys(field)
       .sort((a, b) => field[b] - field[a])
-      .map((k, i) => inflictorWithValue({
-        inflictor: k,
-        value: abbreviateNumber(field[k]),
-        key: i,
-      })) : ''),
+      .map(k => inflictorWithValue(k, abbreviateNumber(field[k]))) : ''),
```

Comment:
It makes no sense since now we have only 2 inputs https://github.com/odota/ui/pull/275/files#diff-525931b3dcf7bf9cbd9e42512129c82dR230